superset-frontend/src/dashboard/components/Dashboard.jsx

```diff
@@ -90,11 +92,18 @@ class Dashboard extends React.PureComponent {
   }
 
   componentDidMount() {
-    const eventData = {};
+    const { dashboardState } = this.props;
+    const eventData = {
+      is_editMode: dashboardState.editMode,
+      mount_duration: Logger.getTimestamp(),
+    };
     const directLinkComponentId = getLocationHash();
     if (directLinkComponentId) {
       eventData.target_id = directLinkComponentId;
     }
+    if (isDashboardEmpty(this.props.layout)) {
+      eventData.is_empty = true;
+    }
```

Comment:
why not do `eventData.is_empty: isDashboardEmpty(this.props.layout)`? Then we could also do this up on line 98 instead too